### PR TITLE
dataset_structure

### PR DESCRIPTION
**Pull request by taraspyndykivskiy**

### CODE REVIEW

File: pyndykivskiytr/dataset_insert_manual.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from dataset_structure import dataset
+import sys
+sys.path.insert(1, 'validator/')
```
Comment (GasperPaul) [inline pyndykivskiytr/dataset_insert_manual.py]: Why is this here?

File: pyndykivskiytr/dataset_insert_manual.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	
+	
+	return dataset
+
+cont=""
+while(cont==""):
+	dataset=insert(dataset)
+	cont=input("Press Enter to continue, other key to leave : ")
+
+# print(dataset)
```
Comment (GasperPaul) [inline pyndykivskiytr/dataset_insert_manual.py]: Plaese, refrain from commiting unnecessary commented code.

File: pyndykivskiytr/lib.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import re
+
+pattern_provider_id=re.compile(r"\d+$")
+pattern_city=re.compile(r"[a-zA-Z -]+$")
+pattern_total_lupa_episodes=re.compile(r"\d+$")
+pattern_average_age=re.compile(r"\d+$")
+
+def provider_id_validator(pattern_provider_id, prompt):
+	number=(validator(pattern_provider_id, prompt))
```
Comment (GasperPaul) [inline pyndykivskiytr/lib.py]: This kind of parentheses use is unnecessary and clatters the code. Stop it.
Also, you need `validator` up top for consistency.

File: pyndykivskiytr/validator/lib.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import re
```
Comment (GasperPaul) [inline pyndykivskiytr/validator/lib.py]: Why you have two versions of this code in different files?

File: pyndykivskiytr/dataset_insert_manual.py (hunk 1 above)
Comment (taraspyndykivskiy) [reply pyndykivskiytr/dataset_insert_manual.py]: Shouldn't we use validator while inserting new data into the dataset (validators for each attribute entered)? Cause that's the reason why I  import  all validation functions into this program.

Comment (GasperPaul) [reply pyndykivskiytr/dataset_insert_manual.py]: You can import without inserting this path into `PATH`. But ok.
